cli/Cargo.toml

```diff
@@ -54,9 +54,16 @@ browser = [
 	"wasm-bindgen",
 	"wasm-bindgen-futures",
 	"browser-utils",
-	"service",
+	"service/light-node",
 ]
 runtime-benchmarks = [ "service/runtime-benchmarks" ]
 trie-memory-tracker = [ "sp-trie/memory-tracker" ]
 full-node = [ "service/full-node" ]
 try-runtime = [ "service/try-runtime" ]
+
+# Configure the native runtimes to use. Polkadot is always enabled by default.
```

Comment:
why not make `polkadot-native` a feature enabled by default? so that it's possible to turn it off

Reply:
Because there needs to be one native runtime currently.

Reply:
Can this work?
```rust
#[cfg(not(any(feature = "polkadot-native", feature = "kusama-native", feature = "westend-native", feature = "rococo-native")))]
compile_error!("at least one native runtime feature must be enabled");
```

Reply:
Yeah this could work, but I don't see the requirement. What we have now works and will be replaced in the near future when the native runtime is gone anyway. 

Then all this stuff will be removed. With this pr you don't even have the chance to not compile all native runtimes, because I don't think that we should support this by now. This would only lead to troubles for peoples trying to run a validator.

Reply:
Good points :-)